presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcConfig.java

```diff
@@ -32,6 +34,7 @@
     private String passwordCredentialName;
     private boolean caseInsensitiveNameMatching;
     private Duration caseInsensitiveNameMatchingCacheTtl = new Duration(1, MINUTES);
+    private UnsupportedTypeHandlingStrategy unsupportedTypeHandlingStrategy = FAIL;
```

Comment:
Do we have already enough data points to determine what the default behavior should actually be?